joomla-projects/privacy-framework
# Make links in action logs notification email clickable

Pull Request for Issue #151 .

### Summary of Changes
This small PR makes links in action logs notification emails clickable. See https://github.com/joomla-projects/privacy-framework/issues/151 for issue description

### Testing Instructions
1. See https://github.com/joomla-projects/privacy-framework/issues/151 to understand how to see the issue
2. Apply patch, test it again and make sure the links in action logs notification email are now clickable.
